src/site/_includes/layout.njk

```diff
@@ -1,5 +1,6 @@
 ---
 show_banner: true
+tags: []
```

Comment:
This removes prior tag inheritance, correct?

Do you need it if we're getting rid of directory data files?

Reply:
Inheritance still works, just through frontmatter and not through data files. Layout.njk is a top-level template, so this is just a fallback tags value for posts that do not define any tags in their frontmatter.

Reply:
Does removing this break anything? I'm not sure if we actually need a tags fallback (the site originally didn't have one) but I added one myself because I needed to reset inheritance at the directory data file level.